software/cnvkit/main.nf

```diff
@@ -0,0 +1,46 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+def options    = initOptions(params.options)
+
+process CNVKIT {
+    tag "$meta.id"
+    label 'process_low'
```

Comment:
Is this the appropriate `label` here @kaurravneet4123 ? Or will this process need more resources by default e.g. `process_medium`?

Reply:
Hi @drpatelh, I think process low if fine as for a wgs data file, it needs only 1.8GB memory. But if you think it is better to change it to 'process_medium', I am happy to do that.